logger.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+// VerbosityLevel maps to a level of logs to be enabled on the command
+type VerbosityLevel uint8
+
+const (
+	// VerbosityLevelOff no log messages displayed
+	VerbosityLevelOff VerbosityLevel = iota
```

Comment:
we could make this map to the equivalent zap values so that we don't need any conversion functions